wire-ios-data-model/Tests/MLS/ProteusToMLSMigrationCoordinatorTests.swift

```diff
@@ -79,7 +79,7 @@ class ProteusToMLSMigrationCoordinatorTests: ZMBaseManagedObjectTest {
         }
 
         // WHEN
-        await sut.migrateOrJoinGroupConversations()
+        try await sut.migrateOrJoinGroupConversations()
```

Comment:
You need to update tests after the latest changes

Reply:
done https://github.com/wireapp/wire-ios/pull/775/commits/0f70ef4d9a3a9dcae13bc0c2519b376c7ba7d0b6

Reply:
@David-Henner There is a SwiftLint error in these tests